tests/Php84/Php84Test.php

```diff
@@ -229,7 +229,12 @@ public function testMbRTrim(string $expected, string $string, ?string $character
 
     public function testMbTrimException()
     {
-        $this->expectException(\ValueError::class);
+        if (PHP_VERSION_ID < 80000) {
+            $this->expectException(\ValueError::class);
+        } else {
+            $this->expectWarning();
+        }
```

Comment:
Shouldn't it be the other way around?
```suggestion
        if (PHP_VERSION_ID < 80000) {
            $this->expectWarning();
        } else {
            $this->expectException(\ValueError::class);
        }
```

Reply:
Yes. I just caught this as well, but you were a bit faster. 🙂